.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
             ${{ runner.os }}-
       - run: make manifests
       - name: Checking if manifests are disaligned
-        run: test -z "$(git diff 2> /dev/null)"
+        run: if [[ $(git diff | wc -l) -gt 0 ]]; then echo ">>> Untracked generated files have not been committed" && git --no-pager diff && exit 1; fi
```

Comment:
Also, this change should be address in a separate commit, something as `chore(ci): output diff files for manifests files`

Reply:
done

Reply:
done